docs/assets/javascript/elements/uxdot-pattern.css

```diff
@@ -1,89 +1,96 @@
+@import url('/styles/rh-code-block-lightdom.css');
+
 :host {
+  container: host / inline-size;
   display: block;
-  container-name: host;
-  container-type: inline-size;
-  margin-block-end: var(--rh-space-2xl, 32px);
+  margin-block-start: var(--rh-space-2xl);
 }
 
 #container {
   display: grid;
-  grid-template-columns: 1fr;
-  grid-template-areas: 'controls' 'example' 'code';
-  gap: var(--rh-space-2xl, 32px) var(--rh-space-4xl, 64px);
-  padding: var(--rh-space-2xl, 32px);
-  border: var(--rh-border-width-sm, 1px) solid var(--rh-color-border-subtle);
-  border-radius: var(--rh-border-radius-default, 3px);
-  background-color: var(--rh-color-surface);
-  color: var(--rh-color-text-primary);
-  border-color: var(--rh-color-border-subtle);
+  grid-template-columns: auto auto;
+  grid-template-areas: 'heading' 'controls' 'content' 'code';
+  gap: var(--rh-space-2xl) var(--rh-space-4xl);
 
-  --rh-color-surface: inherit;
-}
+  @container host (width >= 530px) {
+    grid-template-areas:
+      'heading controls'
+      'content content'
+      'code code';
```

Comment:
```Suggestion
    grid-template-columns: auto auto;
    grid-template-areas:
        "heading controls"
        "content code"
        "content code";    
```
follow up to previous line fix

Reply:
we may way to have a intermediate between 0 and >= 530 px before this to move to 2 columns when the heading and controls can fit on the same line but before we break out to two column for the whole component.